probe-dictionary/explorer.css

```diff
@@ -122,3 +122,8 @@ body {
 .hidden {
     display: none;
 }
+
+.grow:empty:after{
```

Comment:
`.grow` is a very generic class to be hinging this behaviour on. I think `#detail-cpp-guard` may be the better selector for this (though we lose out on getting this behaviour for bug numbers unless we add it specifically)